src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const stylesArr = sourceString.split(';');
+
+  stylesArr.forEach((style) => {
+    const [key, value] = style.split(':');
+
+    if (key && value) {
+      result[key.trim()] = value.trim();
+    }
```

Comment:
You can sicmplify it
```suggestion
    const [key, value] = style.split(':').map(item => item.trim());

    if (key && value) {
      result[key] = value;
    }
```